content/tools/tools.md

```diff
@@ -0,0 +1,50 @@
+---
+date: "10 June 2019"
+title: "The AnVIL Platform"
+author: "AnVIL"
+---
+
+
+# Tools
+
+[[heroSmall]]
+| The AnVIL provides access to an array of analysis and visualization solutions for genomic data analysis. 
+
+Running on Google Cloud Platform (GCP), analysis solutions of the AnVIL are scalable and manageable with GCP, uncapping previous limits of biomedical scalability. Linking billing to user accounts enables AnVIL users to monitor and scale their own analysis. 
+
+The AnVIL uses the Terra analysis engine to launch it’s analysis solutions. These analysis solutions are launched and run within safe workspaces, providing regulated access to high value NHGRI data sets and protecting user loaded data in these workspaces. [AnVIL workspaces](https://support.terra.bio/hc/en-us/articles/360030793091-Terra-FireCloud-Security-Posture) are FISMA moderated and in compliance with standards for working with identifiable human data. 
```

Comment:
it's -> its
the term is FISMA Moderate. It specifies a level of compliance rather than who moderates the compliance